src/Orchard.Web/Modules/Orchard.DynamicForms/Elements/PlaceholderableFormElement.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Orchard.Layouts.Helpers;
+
+namespace Orchard.DynamicForms.Elements {
+    public abstract class PlaceholderableFormElement : LabeledFormElement {
```

Comment:
Actually, I take it back. Not all form elements deriving from `LabeledFormElement` will necessarily have a placeholder (such as drop down lists). So perhaps you were right in the first place by introducing a new sub class.